docs/manual/forms/examples/repeating_groups.rst

```diff
@@ -0,0 +1,114 @@
+.. _examples_repeating_groeps:
+
+================================
+Formulier met herhalende groepen
+================================
+
+In dit voorbeeld maken we een fictief formulier bestaande uit 1 stap met
+een herhalende groep component. We gaan berekeningen laten zien die gebruik maken van de waarden
+die ingevuld zijn in de herhalende groepen.
+
+In dit voorbeeld gaan we er van uit dat u een
+:ref:`formulier met berekeningen <examples_calculations>` kan maken en dat
+u op de hoogte bent van hoe :ref:`logica <manual_logic>` werkt.
+
+Formulier maken
+===============
+
+#. Maak een formulier aan met de volgende gegevens:
+
+   * **Naam**: Herhalende groep demo
+
+#. Klik op het tabblad **Stappen en velden**.
+#. Klik aan de linkerkant op **Stap toevoegen** en selecteer **Maak een nieuwe
+   formulierdefinitie**.
+#. Onder de sectie **(Herbruikbare) stapgegevens** vul het volgende in:
+
+   * **Naam**: Stap met herhalende groep
+
+#. Scroll naar de sectie **Velden** en klik op **Speciale velden**.
+#. Sleep een **Herhalende groep** component op het witte vlak, vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Basis** > **Label**: _`Auto's`
+   * **Weergave** > **Open eerste groep als leeg**: *aangevinkt*
+
+#. Klik op **Formuliervelden** en sleep een **Tekstveld** component binnen de herhalende groep. Vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Label**: Kleur
+
+#. Sleep een **Bedrag** component binnen de herhalende groep. Vul de volgende
+   gegevens in en druk daarna op **Opslaan**:
+
+   * **Label**: Waarde
+
+#. Klik op **Opmaak** en sleep een vrije tekst component op het witte vlak. Vul de volgende gegevens in:
+
+   * Onder **Vrije tekst component**: ``De totale waarde van uw auto's is: €{{ totaleWaarde|localize }}``
```

Comment:
```suggestion
   * Onder **Vrije-tekstcomponent**: ``De totale waarde van uw auto's is: €{{ totaleWaarde|localize }}``
```

Reply:
I think this one refers to the title as it is in the component:
![Screenshot from 2022-08-30 10-33-21](https://user-images.githubusercontent.com/19154114/187389730-602d4e57-7afd-4d26-9b4f-56df8a134de8.png)
So maybe should stay like this?

Reply:
we should fix this component translation, that's the root problem :grimacing: but for now, you can ignore this remark.